packages/web-components/src/components/va-banner/va-banner.tsx

```diff
@@ -70,6 +70,12 @@ export class VaBanner {
    */
   @Prop() dataLabel?: string;
 
+  /**
+   * Optional dissmissedBannerId to allow setting specfic id to be stored in DISMISSED_BANNERS
```

Comment:
Typos here in `dismissedBannerId` and `specific`

Reply:
I can't seem to get specific spelt with all the i's 😆 
```suggestion
   * Optional dissmissedBannerId to allow setting specific id to be stored in DISMISSED_BANNERS
```